### DISCUSSION TITLE
Only allow users with server permissions

### DISCUSSION THREAD

**Discussion by GETrackerDan** [JakyeRU/Larascord#113] [Ideas]
Hi,

Just a suggestion to instead of allowing anybody who is a member of a guild signing into the dashboard.

Check if the user has server permissions such as Moderator, admin, server owner and give them access to the dashboard.

This would serve the purpose for Discord guild staff to have a webpanel to control whatever they're working on vs allowing all users to view the web panel.

**Comment by ValentinNeaga** [comment]
Hello @GETrackerDan,

You can restrict access to the application to users with certain roles. (Check [configuration line 141](https://larascord.jakye.me/configuration#configuration))

Note that the `guilds` and `guilds.members.read` scopes are required for this to work.
